src/commands/run-command.yml

```diff
@@ -12,6 +12,12 @@ parameters:
       script or function, do not specify the file extension. If you specify more than one MATLAB
       command, use a comma or semicolon to separate the commands.
     type: string
+  startup-options:
+    description: >
+      Options passed to MATLAB at startup. If you specify more than one option, use a space to
```

Comment:
Hmm. Pass to MATLAB sounds as if MATLAB is a function. How about these?

1) MATLAB startup options.
2) Options for starting MATLAB.

Reply:
"MATLAB startup options." sounds perfect!